Source/Orts.Simulation/Simulation/Timetables/TTTrain.cs

```diff
@@ -6774,7 +7057,8 @@ public void ProcessTimetableStopCommands(TTTrainCommands thisCommand, int subrou
                 case "wait":
                     if (thisCommand.CommandValues == null)
                     {
-                        Trace.TraceError("Train : {0} : invalid wait command at {1}", Name, thisStationStop == null ? "no station" : thisStationStop.PlatformItem.Name);
+                        Trace.TraceInformation("Train : {0} : invalid wait command at {1}", Name, thisStationStop == null ? "note line" : thisStationStop.PlatformItem.Name);
```

Comment:
This sounds like it should be a warning - something that is wrong in the data, but which we handled (possibly by ignoring it).

Reply:
All items found when processing timetables are consistently set as information.

Reply:
I guess that's something to review later.